src/authorizations/reducers/index.ts

```diff
@@ -24,7 +24,7 @@ import {
   editResource,
 } from 'src/resources/reducers/helpers'
 
-type AuthsState = ResourceState['tokens']
+export type AuthsState = ResourceState['tokens']
```

Comment:
I think we should use `ResourceState['tokens']` in our test code instead of exporting `AuthsState` here